align_server_params : #params defined in align_server_cpp #default values
    align_timeout: 4
    orient_timeout: 2
    x_error_threshold: 0.07
    cargo_error_threshold: 0.05
    orient_error_threshold: 0.03
align_with_terabee_params: #params defined in align_with_terabee.cpp
    cmd_vel_to_pub: 0.7
    distance_bound: 0.07 
    distance_target: 0.45
    cargo_pid_max_distance: 0.5
align_server_topics:
    align_cargo_rocketship:
        enable_x_pub: "hatch_panel_distance_pid/pid_enable"
        x_error_topic: "hatch_panel_distance_pid/pid_debug"
